### PR TITLE
[15.0][ADD] stock_account_no_auto_reconcile

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Prevent to reconcile when posting invoices or validating pickings in PO/SO with posted invoices.

cc @ForgeFlow

### CODE REVIEW

File: stock_account_no_auto_reconcile/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Authors
+~~~~~~~
+
+* ForgeFlow
+* 
+
+Contributors
+~~~~~~~~~~~~
+
+* ForgeFlow S.L. <forgeflow@eficent.com>
```
Comment (DavidJForgeFlow) [inline stock_account_no_auto_reconcile/README.rst]: to update email

File: stock_account_no_auto_reconcile/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* ForgeFlow S.L. <forgeflow@eficent.com>
```
Comment (DavidJForgeFlow) [inline stock_account_no_auto_reconcile/readme/CONTRIBUTORS.rst]: to update email

**Review by DavidJForgeFlow — CHANGES_REQUESTED** [review]
Functional review :+1: code also LGTM!

**Comment by AaronHForgeFlow** [comment]
@DavidJForgeFlow comments attended :)

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM! :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**